### ISSUE TITLE
Default types in help are perhaps superfluous

### ISSUE THREAD

**Issue by eproxus** [max-au/argparse#16]
I'm not a fan of how the default types and values currently are displayed:

```
  -b, --initial-balance Initial balance value, float, 0.0
  -u, --user-email      User email to query, binary
```

1. The comma separated list is hard to parse, especially if there's a comma in the original description text too.
2. The default of `binary` is not really usable since there is no "binary" type in the shell, only "string"

Not sure what the solution here is...

**Comment by max-au** [comment]
One way I can think about it is global switch in options, https://github.com/max-au/argparse/blob/master/src/argparse.erl#L230 that can (a) keep type + default (b) skip them (c) print bases on some template similar to help template

**Comment by max-au** [comment]
Default formatting has been changed to somewhat more readable:
```
-b, --initial-balance    Initial balance value (float, 0.0)
```

**Comment by eproxus** [comment]
The new default looks much better, I think. A switch might be useful too, but at this point I don't think I need it
